src/components/CardList/index.jsx

```diff
@@ -23,7 +22,6 @@ function CardList({
     isFetching = true;
```

Comment:
is not necessary to make a re-fetching of movies, let's keep it simple and just display a good amount of movies so the user can scroll and see some examples of that category